### DISCUSSION TITLE
PCAP Retention possible? What about TrimCAP?

### DISCUSSION THREAD

**Discussion by McSh4dow** [Security-Onion-Solutions/securityonion#2055] [Unsupported Versions] [answered]
Hello,

currently I am running on SO 2.3.2 Standalone with Full PCAP.

Does SO still support TrimCAP for trimming pcaps?

What would happen if the nsm drive capacity is exceeded? Do I need to remove old pcaps by myself or is there a script or cronjob that removes them in a specific interval?

Is there a way to decide what pcaps i would like to collect?

Thanks in advance!
McShadow

**Comment by weslambert** [comment, marked as answer]
We don't support TrimPCAP at this time.  You could consider applying a BPF for Steno: https://docs.securityonion.net/en/latest/bpf.html?highlight=bpf#quoting

**Reply by McSh4dow** [reply]
So there is no way to reduce the amount of pcaps by selecting a specific level or something like that?

I will try using BPF. Thank you for that.

**Reply by TOoSmOotH** [reply]
Steno always makes sure there is at least 10% of /nsm free. It handles cleaning up the disk. BPFs are the best way to limit the traffic that is actually captured. An example would be to not capture say port 443 as 99.9% of this is encrypted. There are pros and cons of doing that but its just an example.

**Reply by TOoSmOotH** [reply]
I am going to drop in an issue to enable users to easily tweak the % free for steno. #2095
